doc/conf.py

```diff
@@ -21,7 +23,28 @@
         "gmt": f">={required_gmt_version}",
     }
 )
+# Is a development version or not.
+isdev = "dev" in __version__ or __version__ == "unknown"
```

Comment:
In the old version, we check if version is dev or not using 
```
len(__version__.split("+")) > 1 or __version__ == "unknown":
```
This new way also works and is easier to read.